src/Microsoft.AspNetCore.Razor.Language/Legacy/CSharpCodeWriter.cs

```diff
@@ -651,5 +656,46 @@ public void Dispose()
             {
             }
         }
+
+        public struct StringLiteralWriter : IDisposable
```

Comment:
The quote is closed in the `Dispose`. In fact I tried to create something similar to the private class `LinePragmaWriter` just above.
It allows to create something like
```
using (var literalWriter = writer.BuildStringLiteralWriter())
{
    literalWriter.Write("foo");
    literalWriter.Write("bar");
}
```